Makefile

```diff
@@ -1,15 +1,17 @@
 all: build
 
 APP_NAME=cluster-ingress-operator
-BIN=cluster-ingress-operator
+OS_OUTPUT_BINPATH=_output/bin
```

Comment:
I don't see the need for an output directory... the only product of the build is a single binary which is already ignored by git

Reply:
If you want the binary on PATH a more conventional approach would be to use `go install`

Reply:
I think that it's a good practice to store the binaries in specific directories to avoid future confusions